### ISSUE TITLE
Error on https://game126092.konggames.com/games/ArmorGames/coinbox-hero/frame/59752757-b7d8-4cb8-a528-a547a84e02ab/

### ISSUE THREAD

**Issue by ernieIzde8ski** [ruffle-rs/ruffle#11277]
# Error Info
Error name: Error
Error message: panicked at 'No uniform for push constant', /home/runner/.cargo/registry/src/index.crates.io-6f17d22bba15001f/wgpu-hal-0.16.0/src/gles/command.rs:725:17
Error stack:
```
Error: panicked at 'No uniform for push constant', /home/runner/.cargo/registry/src/index.crates.io-6f17d22bba15001f/wgpu-hal-0.16.0/src/gles/command.rs:725:17
    at W.n.wbg.__wbg_new_87297f22973157c8 (https://unpkg.com/@ruffle-rs/ruffle/core.ruffle.6392c618460af42080b4.js:1:51142)
    at core::ops::function::FnOnce::call_once{{vtable.shim}}::hf586b8fd5967b71f (wasm://wasm/02c3fd4a:wasm-function[8634]:0x83080c)
    at std::panicking::rust_panic_with_hook::h7f7102b82d51338f (wasm://wasm/02c3fd4a:wasm-function[5629]:0x753fa3)
    at core::panicking::panic_fmt::he4489d678d6570d5 (wasm://wasm/02c3fd4a:wasm-function[8483]:0x82bb79)
    at wgpu_hal::gles::command::<impl wgpu_hal::CommandEncoder<wgpu_hal::gles::Api> for wgpu_hal::gles::CommandEncoder>::set_push_constants::h5db2ee8b52bb113a (wasm://wasm/02c3fd4a:wasm-function[3425]:0x64e776)
    at <T as wgpu::context::DynContext>::command_encoder_end_render_pass::hdd7f140e85dd1100 (wasm://wasm/02c3fd4a:wasm-function[8017]:0x81663a)
    at <wgpu::RenderPass as core::ops::drop::Drop>::drop::haa1d87d2f8884672 (wasm://wasm/02c3fd4a:wasm-function[6779]:0x7aa61b)
    at <ruffle_render_wgpu::backend::WgpuRenderBackend<T> as ruffle_render::backend::RenderBackend>::apply_filter::h890260d16e5ff0ca (wasm://wasm/02c3fd4a:wasm-function[1336]:0x3cc829)
    at ruffle_core::avm2::globals::flash::display::bitmap_data::apply_filter::h051327b920d31198 (wasm://wasm/02c3fd4a:wasm-function[1170]:0x37a894)
    at ruffle_core::avm2::function::Executable::exec::h6950dbf1ee2f272a (wasm://wasm/02c3fd4a:wasm-function[807]:0x28e107)
```
AVM2 stack:
```
    at flash.display::BitmapData/flash::display::BitmapData::applyFilter()
    at BlurStill/blur()
```

# Player Info
Allows script access: true
Renderer: wgpu
Adapter Backend: Gl
Adapter Name: "ANGLE (NVIDIA Corporation, NVIDIA GeForce RTX 2060 SUPER/PCIe/SSE2, OpenGL 4.5.0)"
Adapter Device Type: Other
Adapter Driver Name: ""
Adapter Driver Info: ""
Enabled features: Features(TEXTURE_ADAPTER_SPECIFIC_FORMAT_FEATURES | PUSH_CONSTANTS)
Available features: Features(TEXTURE_COMPRESSION_BC | CLEAR_TEXTURE | MULTIVIEW)
Current limits: Limits { max_texture_dimension_1d: 32768, max_texture_dimension_2d: 32768, max_texture_dimension_3d: 16384, max_texture_array_layers: 256, max_bind_groups: 4, max_bindings_per_bind_group: 640, max_dynamic_uniform_buffers_per_pipeline_layout: 8, max_dynamic_storage_buffers_per_pipeline_layout: 0, max_sampled_textures_per_shader_stage: 16, max_samplers_per_shader_stage: 16, max_storage_buffers_per_shader_stage: 0, max_storage_textures_per_shader_stage: 0, max_uniform_buffers_per_shader_stage: 11, max_uniform_buffer_binding_size: 16384, max_storage_buffer_binding_size: 0, max_vertex_buffers: 8, max_buffer_size: 268435456, max_vertex_attributes: 16, max_vertex_buffer_array_stride: 255, min_uniform_buffer_offset_alignment: 256, min_storage_buffer_offset_alignment: 256, max_inter_stage_shader_components: 60, max_compute_workgroup_storage_size: 0, max_compute_invocations_per_workgroup: 0, max_compute_workgroup_size_x: 0, max_compute_workgroup_size_y: 0, max_compute_workgroup_size_z: 0, max_compute_workgroups_per_dimension: 0, max_push_constant_size: 96 }
Surface quality: high
Surface samples: 4
Surface size: Extent3d { width: 550, height: 400, depth_or_array_layers: 1 }

# Page Info
Page URL: https://game126092.konggames.com/games/ArmorGames/coinbox-hero/frame/59752757-b7d8-4cb8-a528-a547a84e02ab/?kongregate_host=www.kongregate.com
SWF URL: https://game126092.konggames.com/gamez/0012/6092/live/CoinsKong_secure.swf?kongregate_game_version=1317428716

# Browser Info
User Agent: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/112.0.0.0 Safari/537.36
Platform: Linux x86_64
Has touch support: false

# Ruffle Info
Version: 0.1.0
Name: nightly 2023-05-29
Channel: nightly
Built: 2023-05-29T00:19:07.550Z
Commit: 0955ab40e67d4668f3ebd5fbdf3602750e35abb8
Is extension: false

# Metadata
width: 800
height: 500
frameRate: 30
numFrames: 3
swfVersion: 10
backgroundColor: undefined
isActionScript3: true
uncompressedLength: 5649914

**Comment by ernieIzde8ski** [comment]
This occurs after the initial cutscene when trying to enter the shop on the left

**Comment by n0samu** [comment]
Looks like the same problem as #11064

**Comment by n0samu** [comment]
Should be fixed by #11753 and #11750
